api/v1alpha1/nnfstorageprofile_types.go

```diff
@@ -115,6 +120,12 @@ type NnfStorageProfileLustreData struct {
 
 	// OstOptions contains options to use for libraries used for an OST target.
 	OstOptions NnfStorageProfileLustreMiscOptions `json:"ostOptions,omitempty"`
+
+	// MountRabbit specifies mount options for mounting on the Rabbit.
+	MountRabbit string `json:"mountRabbit,omitempty"`
+
+	// MountCompute specifies mount options for mounting on the Compute.
+	MountCompute string `json:"mountCompute,omitempty"`
 }
 
 // NnfStorageProfileCmdLines defines commandlines to use for mkfs, and other utilities.
```

Comment:
Update this comment to clarify that these are on the non-lustre commandlines.  I had to go back and forth between these two structs a few times to identify the difference between the two structs that define MountRabbit/MountCompute.

Reply:
whoops I pushed this to the wrong branch. I'm still working on these changes.

Reply:
okay it's been tested now. I fixed the two things you commented on so far